drwpow/openapi-typescript
# feat: parameters is optional when every parameter is optional

Closes #1334

## Changes

Makes `parameters` type optional `parameters?: {` when every parameter is optional.

https://github.com/drwpow/openapi-typescript/issues/1334

## How to Review

* Verify APIs with *any* required parameters continue to generate non-optional `parameters: {`
* Verify APIs with *no required* parameters generates optional `parameters?: {`

I haven't tested this beyond the single `optionalParamSchema` already present in the test corpus.

## Checklist

- [ ] `pnpm run update:examples` run (only applicable for openapi-typescript)
